openedx/adg/lms/applications/models.py

```diff
@@ -30,12 +36,43 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
+    """
+    Model to save the business lines offered by ADG.
+    """
+    title = models.CharField(max_length=255, )
+    logo = models.ImageField(
+        upload_to='business-lines/uploaded-logo/', null=True,
+        validators=[FileExtensionValidator(LOGO_ALLOWED_EXTENSION), validate_logo_size],
+    )
+    description = models.TextField()
+
+    def __str__(self):
+        return 'Business Line {}'.format(self.title)
+
+
 class UserApplication(TimeStampedModel):
     """
     Model for status of all required parts of user application submission.
     """
     user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
     city = models.CharField(verbose_name=_('City'), max_length=255, )
+
+    saudi_national = models.BooleanField(default=True)
+    MALE = 'male'
+    FEMALE = 'female'
+    OTHER = 'other'
+
+    GENDER_CHOICES = (
+        (MALE, _('Male')),
+        (FEMALE, _('Female')),
+        (OTHER, _('Prefer not to answer')),
+    )
+    gender = models.CharField(choices=GENDER_CHOICES, max_length=8, default=OTHER,)
+    phone_number = PhoneNumberField(blank=True, region='SA')
+    birth_date = models.DateField(default=timezone.now)
```

Comment:
For education and work, we have created separate fields for month and year. Here as per design the day/month/year are separate fields.

@ahmed-faraz46 Should we split `birth_date` or is this fine. If we go with this then we will have to customize it on the front end (Django form).

Reply:
I guess because here we have the day as well, so it makes sense to keep it. More I'd suggest to use some JS datepicker to select dates. Here is an [option](https://www.jqueryscript.net/demo/dropdown-date-picker/). It would be so easy to tackle the complexities on changing the options of days drop down according to the selected months. And when on front end we will use it. in the backend we will receive a date in string format and we can convert it to python simply.